src/messages/submessages/info_reply.rs

```diff
@@ -10,12 +10,12 @@ pub struct InfoReply {
   /// Indicates an alternative set of unicast addresses that
   /// the Writershould use to reach the Readers when
   /// replying to the Submessages that follow.
-  pub unicast_locator_list: LocatorList,
+  pub unicast_locator_list: Vec<Locator>,
 
   /// Indicates an alternative set of multicast addresses that the Writer
   /// should use to reach the Readers when replying to the Submessages that
   /// follow.
   ///
   /// Only present when the MulticastFlag is set.
-  pub multicast_locator_list: Option<LocatorList>,
+  pub multicast_locator_list: Option<Vec<Locator>>,
```

Comment:
@jhelovuo is there a semantic difference here between `None` or an empty `Vec`? if not, can this be flattened?

Reply:
Yes there is: It serializes differently.

Removing this distinction may break serialization. See RTPs spec "9.4.5.10 InfoReply Submessage".